### DISCUSSION TITLE
Div centered automatically

### DISCUSSION THREAD

**Discussion by monkeydmao** [vuejs/vitepress#508] [Q&A]
Im using vitepress for the first time and on my home page when im creating a div with a vue component, the dive is automatically centered and cant take the full wisth of the screen (view the screen)

![image](https://user-images.githubusercontent.com/76960779/151188718-a0fee6b6-60c5-4877-8de5-8a055c5706b7.png)

how can i fix that please??

**Comment by iomariani** [comment]
Just add a custom CSS changing `.container` `max-width` to your preference:

https://github.com/vuejs/vitepress/blob/45b65ce8b63bd54f345bfc3383eb2416b6769dc9/src/client/theme-default/components/Page.vue#L37-L41

**Reply by xu4wang** [reply]
Just notice this is the issue I'm facing too. #1709 

@iomariani would you please help clarify how to customize the max-width? 
I am not sure should I follow https://vitepress.vuejs.org/guide/theme-introduction#customizing-css , since it mentions only [default theme CSS variables](https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css) can be overridden.

**Reply by iomariani** [reply]
> Just notice this is the issue I'm facing too. #1709 
> 
> 
> 
> @iomariani would you please help clarify how to customize the max-width? 
> 
> I am not sure should I follow https://vitepress.vuejs.org/guide/theme-introduction#customizing-css , since it mentions only [default theme CSS variables](https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/styles/vars.css) can be overridden.  

Yes, just follow those steps to override the default css applied.
